examples/swap/scripts/localnet.sh

```
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+if [ "$1" = "start" ]; then npx hardhat localnet --exit-on-error & sleep 10; fi
+
+echo -e "\n🚀 Compiling contracts..."
+npx hardhat compile --force --quiet
+
+ZRC20_ETHEREUM=$(jq -r '.addresses[] | select(.type=="ZRC-20 ETH on 5") | .address' localnet.json)
```

Shellcheck

[warning] ZRC20_ETHEREUM appears unused. Verify use (or export if used externally).

(SC2034)